site/content/getting-started/index.md

```diff
@@ -185,7 +185,7 @@ The Heads Up Display (HUD) is a new an innovative interface that provides access
 
 {{< img "images/hud-welcome.png" >}}
 
-The HUD is overlayed on top of the target application in your browser when enabled via the ‘Manual Explore’ screen or toolbar option. Only modern browsers such as Firefox and Chrome are supported.
+The HUD is overlayed on top of the target application in your browser when enabled via the ‘Manual Explore’ screen or toolbar option. Only modern browsers such as Firefox and Chrome are supported. Firefox users may need to [disable "Enhanced Tracking Protection"](https://www.zaproxy.org/getting-started/images/firefox-protection-hud.png) in order to be able to see the HUD interface. This can be done by clicking on the shield icon in in the URL bar.
```

Comment:
```suggestion
The HUD is overlayed on top of the target application in your browser when enabled via the ‘Manual Explore’ screen or toolbar option. Only modern browsers such as Firefox and Chrome are supported. Firefox users may need to [disable "Enhanced Tracking Protection"](https://www.zaproxy.org/getting-started/images/firefox-protection-hud.png) in order to be able to see the HUD interface. This can be done by clicking on the shield icon in the URL bar.
```

Reply:
Also, please use relative URLs, ie "/getting-started/images/firefox-protection-hud.png" (removing the `https://www.zaproxy.org`)  - that way they work sorrectly when we run the website locally 😁

Reply:
Should use relative link for the image.

Reply:
Can be simplified more `images/firefox-protection-hud.png`.